FreeRTOS-Plus/Source/Application-Protocols/network_transport/mbedtls_pk_pkcs11.c

```diff
@@ -92,7 +110,7 @@ static void * p11_ecdsa_ctx_alloc( void );
  * @param xPkHandle The CK_OBJECT_HANDLE for the target private key.
  * @return CKR_OK on success
  */
-static CK_RV p11_ecdsa_ctx_init( void * pvCtx,
+static CK_RV p11_ecdsa_ctx_init( mbedtls_pk_context * pxMbedtlsPkCtx,
```

Comment:
There was a change to these functions to take in the `mbedtls_pk_context` instead of taking in `mbedtls_pk_context->pk_ctx` link to the [PR](https://github.com/Mbed-TLS/mbedtls/pull/7463)